contents/admin/3-accounts.rst: add steps for listing users, permissions (#210)
datajoint/datajoint-docs: contents/admin/3-accounts.rst

 CREATE USER 'alice'@'%' IDENTIFIED BY 'alices-secret-password';

Existing users can be listed using the following SQL:

.. code-block:: mysql

>  SELECT User,Host from mysql.user; 

Reviewer: ```suggestion
 SELECT user, host FROM mysql.user; 
```

Author: merged. `User` and `Host` match the actual column definitions, but yes, lower case is more straightforward and less confusing since MySQL doesn't actually care about this.

To note, the :code:`ALL PRIVILEGES` option allows the user to create
and remove databases without administrator intervention.

> Once created, grants for a user can be listed using the :code:`SHOW GRANTS`

Reviewer: ```suggestion
Once created, user privileges can be listed using the :code:`SHOW GRANTS`
```

Author: merged.
